src/app/logic/processGame.js

```diff
@@ -0,0 +1,146 @@
+import Answers from '../components/Answers';
+import VisualImage from '../components/VisualImage';
+import { getQuestion, initGame, isGameInitialized } from './quiz';
+
+class Game {
+  getTimeLeft() {
+    return this.timeLeft;
+  }
+
+  getMode() {
+    return this.mode;
+  }
+
+  getRunning() {
+    return this.running;
+  }
+
+  getQuestions() {
+    return this.questions;
+  }
+
+  reduceTime() {
+    this.timeLeft -= 0.5;
+  }
+
+  addQuestion(question) {
+    this.questions.push(question);
+  }
+
+  newGame(mode) {
```

Comment:
since it is a function, how about startNewGame? So the function name starts with a describing verb?

Reply:
startNewGame can be misleading. This function only prepare instance to start a new game

Reply:
I will rename this function to prepareNewGame